### PR TITLE
PLNSRVCE-990: Use java cache when it's available

### PR DESCRIPTION
**Pull request by Michkov**
Set maven mirror when cache service is available.

### CODE REVIEW

File: task/buildah/0.1/buildah.yaml [hunk 1]
```diff
@@ -90,6 +90,11 @@ spec:
         echo "Cannot find Dockerfile $(params.DOCKERFILE)"
         exit 1
       fi
+      if [ -n "$JVM_BUILD_WORKSPACE_ARTIFACT_CACHE_PORT_80_TCP_ADDR" ] && grep -q '^RUN \(./\)\?mvn' "$dockerfile_path"; then
```
Comment (psturc) [inline task/buildah/0.1/buildah.yaml:93]: just wondering, where is this env var coming from?

Comment (Michkov) [reply task/buildah/0.1/buildah.yaml:93]: Kubernetes creates it out of box

**Comment by psturc** [comment]
```
STEP-S2I-GEN

Generated Env file
------------------------------
MAVEN_CLEAR_REPO=true
MAVEN_MIRROR_URL=http://172.30.198.99/v1/cache/default/0/
------------------------------
Application dockerfile generated in /gen-source/Dockerfile.gen
```

verified

**Review by psturc — APPROVED** [review]

File: task/s2i-java/0.1/s2i-java.yaml [hunk 2]
```diff
@@ -143,7 +145,7 @@ spec:
       name: varlibcontainers
     securityContext:
       runAsUser: 0
-  - image: registry.redhat.io/ubi8/python-39:1-90.1669637098
+  - image: registry.redhat.io/ubi8/python-39:1-87.1669838026
```
Comment (tkdchen) [inline task/s2i-java/0.1/s2i-java.yaml:148]: @Michkov What is this changed for?

Comment (Michkov) [reply task/s2i-java/0.1/s2i-java.yaml:148]: I put there wrong unexisting tag in PR before so this was a fix

**Pull request merged**